nixos/tests/hostname.nix

```diff
@@ -39,9 +38,13 @@ let
         # Test if NixOS computes the correct FQDN (either a FQDN or an error/null):
         assert "${getStr nodes.machine.networking.fqdn}" == "${getStr fqdnOrNull}"
 
+        # This is returned by net tools (hostname, domainname) when domain is missing
+        emptyDomainNamePlaceholder="(none)"
+
         # The FQDN, domain name, and hostname detection should work as expected:
-        assert "${fqdn}" == machine.succeed("hostname --fqdn").strip()
-        assert "${optionalString (domain != null) domain}" == machine.succeed("dnsdomainname").strip()
+        assert "${hostName}" == machine.succeed("hostname").strip()
```

Comment:
Is there no --fqdn parameter anymore?

Reply:
When you enter a fresh NixOS VM and run these commands, this is what I get:
![2023-09-30-170941_298x199_scrot](https://github.com/NixOS/nixpkgs/assets/61136584/03b836ff-6117-44ce-93ce-009d5711346a)
Which is a different behaviour to what used to be in the tests assertions.